lib/cn/cn.c

```diff
@@ -957,6 +957,10 @@ cn_open(
     cn_tree_foreach_leaf(tn, cn->cn_tree) {
         cn_tree_node_get_max_key(tn, kbuf, sizeof(kbuf), &klen);
 
+        /* TODO: Handle a ptomb-only node
```

Comment:
This is what you created a ticket for?

Reply:
Yes, I will be creating a ticket for this.

Reply:
Might be worth including the ticket number in the comment when you do.